core/src/main/java/org/aerogear/mobile/core/ServiceModule.java

```diff
@@ -1,15 +1,4 @@
 package org.aerogear.mobile.core;
 
-import org.aerogear.mobile.core.configuration.ServiceConfiguration;
-
-
 public interface ServiceModule {
```

Comment:
[Optional] This interface may no longer be needed.

Reply:
Yeap, maybe in the future, let's leave it for now only to _mark_ the class is a _Service_